bloom-housing/bloom
# fix: use appropriate copy for FCFS vs Lottery in Application Confirmation screen

# Pull Request Template

## Issue Overview

This PR addresses #2510 


## Description

In QA testing [it was discovered](https://app.zenhub.com/workspaces/bloom-5dc32d7144bd400001315dac/issues/bloom-housing/bloom/2510#issuecomment-1109969397) the original PR #2558 only included copy for First Come First Serve listings. This updates the application confirmation screen so that Lottery listings will also show the correct copy.

## How Can This Be Tested/Reviewed?

The [deploy preview](https://deploy-preview-2702--dev-bloom.netlify.app/) has an outdated DB seed so the two listing types will need to be tested locally.

Fill out an application for both types and see if the final confirmation screen shows either:

* Eligibile applicants will be contacted by on a **first come first serve** basis until vacancies are filled
* The lottery will be held on **(DATE)**. Eligible applicants will be contacted by the agent in lottery rank order until vacancies are filled

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
